components/MemoryGrid/index.js

```diff
@@ -59,6 +60,7 @@ export default function MemoryGrid({
                 }, 600);
               }}
               onDragStart={(event) => event.preventDefault()}
+              role="button"
               aria-label="conceiled card"
```

Comment:
I was just wondering, did you mean concealed instead of conceiled? Otherwise everything looks fine! Let's get those cards discovered! 🚀